qa/common/gen_qa_models.py

```diff
@@ -870,3 +915,19 @@ def create_fixed_models(
                       (8,-1), (8,4), (8,4), 32)
         create_models(FLAGS.models_dir, np_dtype_string, np.int32, np_dtype_string,
                       (-1,8,-1), (2,8,2), (2,8,2), 32)
+
+    if FLAGS.ensemble:
```

Comment:
Shouldn't this be indented in so that it is inside "if FLAGS.variable"?

Reply:
These nops are used in both variable and non-variable size test. For the non-variable size test, the ensemble models are fixed size, but internally it will pass the tensors to variable size nop with the same dims size, so that we don't need to create fixed size version of the nops.

Reply:
ok